src/routeGeneration/templateHelpers.ts

```diff
@@ -381,6 +381,14 @@ export class ValidationService {
         return this.validateEnum(name, value, fieldErrors, enums, parent);
       }
 
+      if (!(value instanceof Object)) {
+        fieldErrors[parent + name] = {
+          message: `Not an Object`,
```

Comment:
The error message of `Not an object` is not very descriptive. If I was a consumer of an API that returned that error message, I would be quite confused. Maybe it could simply be `Expected an object for this property but received ${typeof value} instead`.

Reply:
I agree. Since the other validations are rather brief as well, examples include "invalid boolean value", "invalid array" and "invalid string value", I think there is an opportunity to add some more convenience that may be shared.
For now, is it fine if I were to be closer to the other messages and return 'invalid object'?

Reply:
Not a problem. I’m happy to get this moving to help you with your other PRs. :)

Reply:
Not a problem. I’m happy to get this moving to help you with your other PRs. :)

Reply:
I'll create an issue and revisit this, thanks for accepting a mediocre message for now.